src/super_gradients/training/sg_model/sg_model.py

```diff
@@ -299,6 +319,50 @@ def build_model(self,  # noqa: C901 - too complex
 
         self._load_checkpoint_to_model()
 
+    def _deactivate_quant_modules_wrapping(self):
+        """
+        Deactivates quant modules wrapping, so that further modules won't use Q/DQ layers.
+        """
+        if self.use_quant_modules:
+            quant_modules.deactivate()
+
+    def _initialize_quant_modules(self):
+        """
+        Initialize quant modules wrapping.
+        """
+        self.use_quant_modules = core_utils.get_param(self.arch_params, "use_quant_modules", False)
+        self.quant_modules_calib_method = core_utils.get_param(self.arch_params, "quant_modules_calib_method", "percentile")
+        self.per_channel_quant_modules = core_utils.get_param(self.arch_params, "per_channel_quant_modules", False)
+
+        if self.use_quant_modules:
+            if _imported_pytorch_quantization_failure is not None:
```

Comment:
can we maybe call a function that does the imports?
I know it is not that conventional, but I don't want the warning to appear on every SG train. instead, lets load it only if QAT is requested